packages/features/form-builder/FormBuilderField.tsx

```diff
@@ -54,6 +54,7 @@ export const FormBuilderField = ({
   const { t } = useLocale();
   const { control, formState } = useFormContext();
 
+  const labelBeforeNormalize = field.label;
```

Comment:
Why are we using the `labelBeforeNormalize`? What is the difference here?

Reply:
This is required to respect the custom label entered for location in BookingQuestions, because the `getAndUpdateNormalizedValues()` fn will modify the field.label to default label `location`.

Reply:
I have fixed the logic so that we use field.label only. The issue was because we were setting field.label to empty string for radioInput type field(e.g. location) when it has just one option and that is an input.

Now, we are not modifying that, and introduced a separate property noLabel and doing some prop drilling.